dazel.py

```diff
@@ -192,11 +192,14 @@ def _fix_win_symlink(self, docker_exec_command):
         for path in list(p.glob('bazel-*')):
             command = "%s realpath %s" % (
                 docker_exec_command, str(path.name))
-            output = self._run_command(command).strip()
-            drive = pathlib.PureWindowsPath(path.drive)
-            local_directory = drive.joinpath(pathlib.PurePosixPath(output))
-            path.unlink()
-            path.symlink_to(local_directory, target_is_directory=True)
+            try:
+                output = self._run_command(command).strip()
+                drive = pathlib.PureWindowsPath(path.drive)
+                local_directory = drive.joinpath(pathlib.PurePosixPath(output))
+                path.unlink()
+                path.symlink_to(local_directory, target_is_directory=True)
+            except subprocess.CalledProcessError:
+                logger.info("INFO: Skipping fixing symlink, it already exists.")
```

Comment:
```suggestion
            try:
                output = self._run_command(command).strip()
            except subprocess.CalledProcessError:
                logger.info("INFO: Skipping fixing symlink, it already exists.")
            else:
                drive = pathlib.PureWindowsPath(path.drive)
                local_directory = drive.joinpath(pathlib.PurePosixPath(output))
                path.unlink()
                path.symlink_to(local_directory, target_is_directory=True)
```

Reply:
I have updated according to your comments now and tested.

Reply:
@mishas . Ping ...